parlai/crowdsourcing/tasks/fast_acute/analysis.py

```diff
@@ -0,0 +1,742 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+"""
+ACUTE-Eval Analyzer.
+
+FOR ANALYSIS!!
+"""
+
+import hashlib
+import json
+import os
+from typing import Dict, Any, List, Optional
+
+import numpy as np
+import pandas as pd
+from IPython.core.display import HTML
+from mephisto.tools.data_browser import DataBrowser as MephistoDataBrowser
+from mephisto.abstractions.databases.local_database import LocalMephistoDB
+from mephisto.data_model.unit import Unit as MephistoUnit
+from scipy.stats import binom_test
+
+from parlai.core.params import ParlaiParser
+from parlai.crowdsourcing.tasks.acute_eval.acute_eval_blueprint import BLUEPRINT_TYPE
+
+# To register the ACUTE-Eval blueprint
+from parlai.crowdsourcing.tasks.fast_acute.util import get_hashed_combo_path
+
+_ = BLUEPRINT_TYPE
+# NOTE: BLUEPRINT_TYPE needs to be imported here to register the blueprint
+
+# throw away turkers below this threshold
+AGREEMENT_THRESHOLD = 0.8
+# do we count ties as agreements?
+AGREEMENT_TIES_OKAY = False
+
+# Prepended to checkbox columns in self.dataframe
+CHECKBOX_PREFIX = 'checkbox: '
```

Comment:
Nit - would you also want to add these as configuration options? Not necessary, of course.

Reply:
Hmm, they definitely could be, but in general I suppose we could hold off on adding flags such as these until there's a demonstrated need, in order to avoid cluttering up the code prematurely. I've added a note to indicate that these could be made into flags, though